solutions/instances/main.tf

```diff
@@ -162,8 +151,8 @@ module "scc" {
   en_instance_crn                   = var.existing_en_crn
   skip_cos_iam_authorization_policy = var.skip_scc_cos_auth_policy
   resource_tags                     = var.scc_instance_tags
-  attach_wp_to_scc_instance         = var.provision_scc_workload_protection
-  wp_instance_crn                   = var.provision_scc_workload_protection ? module.scc_wp[0].crn : null
+  attach_wp_to_scc_instance         = var.provision_scc_workload_protection && var.existing_scc_instance_crn == null
```

Comment:
This means that user cannot attach workload protection instance to an existing SCC instance. @in-1911 I think we talked about this use case. Do we want to support that or not?